Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -696,7 +671,7 @@ protected function requireNodeAggregateToOccupyDimensionSpacePoint(
     ): void {
         if (!$nodeAggregate->occupiesDimensionSpacePoint($originDimensionSpacePoint)) {
             throw new DimensionSpacePointIsNotYetOccupied(
-                'Dimension space point ' . json_encode($originDimensionSpacePoint)
+                'Dimension space point ' . json_encode($originDimensionSpacePoint, JSON_THROW_ON_ERROR)
```

Comment:
```suggestion
                'Dimension space point ' . json_encode($originDimensionSpacePoint, JSON_PARTIAL_OUTPUT_ON_ERROR)
```

i guess `JSON_PARTIAL_OUTPUT_ON_ERROR` would be better but this is only for linting purposes so yolo